Automattic/jetpack
# Search dashboard: add scaffolding for Record Meter

#### Changes proposed in this Pull Request:
Add scaffolding for Record Meter feature in wp-admin's search dashboard.

Todo:

#### Jetpack product discussion
pcNPJE-iq-p2

#### Does this pull request change what data or activity we track or use?
No.

#### Testing instructions:
On a site with Instant Search enabled, load the search dashboard with `features=record-meter` in the query string:

`wp-admin/admin.php?page=jetpack-search&features=record-meter`

You should see the 'Your search records' title like shown:

<img width="969" alt="Screen Shot 2022-01-28 at 17 56 56" src="https://user-images.githubusercontent.com/17325/151489888-80b63a67-2b3a-42cd-8073-bc515b8c95fb.png">

Now try with some old nonsense in the `features` param, like `spaniels`, or omit the param altogether.

`wp-admin/admin.php?page=jetpack-search&features=spaniels`

You should not see the 'Your search records' title at all:

<img width="623" alt="Screen Shot 2022-01-28 at 17 57 11" src="https://user-images.githubusercontent.com/17325/151489967-14c8f701-2e2a-47e9-b0c6-e4c1adf7d8df.png">

cc @annacmc
